### DISCUSSION TITLE
If i assign photos from "Show All New Faces" Section.. does it learn?

### DISCUSSION THREAD

**Discussion by danmed** [photoprism/photoprism#2166] [Q&A] [answered]
Hopefully this is my last question for a while :)

There's thousands of photos with faces but they aren't clustered (this is normal i believe).. i've gone through and marked maybe a hundred or so of them to specific people.. Will this enable Photoprism to recognise some of the others? Like by me adding more photos to people does it learn to march more faces?

**Comment by graciousgrey** [comment, marked as answer]
Yes, that's expected! 
When you are adding names to faces, PhotoPrism updates face clusters in the background and it's possible that some of the unnamed faces can be matched to a person automatically :)

**Reply by danmed** [reply]
Should i re-run the face index or will it sort things out itself?

**Reply by graciousgrey** [reply]
It will do this automatically in the background
